optin/puller/puller/db/InProgressOptinDatabase.py

```diff
@@ -77,6 +77,13 @@ def max_processed_height(self):
 		max_request_height = cursor.fetchone()[0]
 		return max(max_error_height or 0, max_request_height or 0)
 
+	@property
+	def requests(self):
+		"""Returns requests."""
+		cursor = self.connection.cursor()
+		cursor.execute('''SELECT * FROM optin_request ORDER BY transaction_height''')
+		return cursor
```

Comment:
why returning cursor instead of addresses?

Reply:
nicer iteration? (iterator vs collection)

Reply:
i mean you could just yield and not return tuples ^^